src/components/calcite-color-picker/calcite-color-picker.e2e.ts

```diff
@@ -1081,4 +1081,29 @@ describe("calcite-color-picker", () => {
     await assertHiddenSection(["channels"]);
     await assertHiddenSection([]);
   });
+
+  it("allows editing handles via keyboard", async () => {
+    const page = await newE2EPage({
+      html: `<calcite-color-picker value="#beefee"></calcite-color-picker>`
+    });
+
+    const picker = await page.find("calcite-color-picker");
+    const spy = await picker.spyOnEvent("calciteColorPickerChange");
+
+    await page.keyboard.press("Tab");
+    await page.keyboard.press("ArrowUp");
+    expect(spy).toHaveReceivedEventTimes(1);
```

Comment:
This test should also assert the color to make sure the arrow keys (and shift modifier) update values in the expected direction.

Reply:
working on this right now, fyi